src/data/nat/factorial.lean

```diff
@@ -123,4 +123,24 @@ begin
   { exact add_factorial_lt_factorial_add _ hi }
 end
 
+lemma add_factorial_le_factorial_add (i : ℕ) (n : ℕ) :
+  i + (n + 1)! ≤ (i + n + 1)! :=
```

Comment:
Does it make things harder for you if this statement is 
```suggestion
  i + (n + 1)! ≤ (i + (n + 1))! :=
```
That makes it match the `.succ` name I suggest

Reply:
Isn't this the same?  Parentheses associate like this by default, right?

Reply:
I think by coincidence its true by definition:
```
example (a b : ℕ) : (a + b + 1) = (a + (b + 1)) := rfl
```

Reply:
That's great: I can state it the way you said and use it the way I had it!